app/helpers/application_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ApplicationHelper
+# Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = "Food App"
```

Comment:
You could check for `present?` instead of `empty?` here and not have a default page_title.

You can try this in the console, but it should be something like:
```
[21] pry(main)> nil.present?
=> false
[22] pry(main)> ''.present?
=> false
[23] pry(main)> ' '.present?
=> false
[24] pry(main)> 's'.present?
=> true
```